Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml

```diff
@@ -90,6 +90,11 @@
     ClothingOuterHICorporateJacket: 2 # Einstein Engines - Clothing addition
```

Comment:
`ClothingOuterEeCorporateJacket`
`ClothingOuterHiCorporateJacket`
`ClothingOuterHmCorporateJacket`
YML linter fails the tests because this field is fully case sensitive, and the second letter in these acronyms isn't capitalized in Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

Reply:
oh shit i though i fixed this already, thank you.